### DISCUSSION TITLE
curl_multi_poll: `revents` field supported?

### DISCUSSION THREAD

**Discussion by nojb** [curl/curl#11748] [General]
In `multi.h`, the `revents` field is marked as not supported:

https://github.com/curl/curl/blob/458c70db6a8e51e6f47313ac91da736cfe8d3fbb/include/curl/multi.h#L121

Is this comment accurate? In my tests with the version of libcurl in Ubuntu 20.04 it doesn't seem to work, but a quick glance of the latest source seems to indicate it should be updated correctly.

Any insights?

**Comment by bagder** [comment]
I believe 6d30f8ebe made them populated. The comment is seriously outdated.

**Reply by nojb** [reply]
OK, thanks for the confirmation!
